pgmpy/tests/test_estimators/test_expert.py

```diff
@@ -272,8 +278,77 @@ def test_edge_orientation_priority(self):
             orientations=orientations,
             effect_size_threshold=0.0001,
             show_progress=False,
+            descriptions=self.descriptions,
         )
 
         # Check that specified orientations take precedence
         if ("Income", "Education") in dag.edges():
             assert ("Education", "Income") not in dag.edges()
+
+    @unittest.skipUnless(
```

Comment:
Could you also please add tests for the `root_node` in some other causal discovery algorithms like PC, Hill-Climb Search ?